pontoon/sync/tests/test_vcs_models.py

```diff
@@ -297,11 +334,10 @@ def setUp(self):
 
         # Make sure VCSConfiguration instance is initialized
         self.db_project.configuration_file = self.toml
-        self.vcs_project = VCSProject(self.db_project, locales=[self.locale])
-
-        self.vcs_project.configuration.configuration_path = os.path.join(
-            PROJECT_CONFIG_CHECKOUT_PATH, self.db_project.configuration_file,
+        self.db_project.source_repository.permalink_prefix = (
+            "https://example.com/{locale_code}"
```

Comment:
I'd remove `{locale_code}` to avoid confusion: we don't use {locale_code} in projects that use configuration files.